MAiD - Case Management App/force-app/main/default/permissionsets/ICY_Guest_User_Permission.permissionset-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <hasActivationRequired>false</hasActivationRequired>
+    <label>ICY Guest User Permission</label>
+</PermissionSet>
```

Comment:
This permission set has no content. If empty it may be better to not include this in git as it may strip the values in the actual permission set in Prodution @lilianenassar-bib

Reply:
@cgijeffolsen I agree